src/RoleObjectFactory.js

```diff
@@ -117,6 +117,7 @@ function createAccessibilityObjectForRole(config) {
     case ROLES.SEARCH:
     case ROLES.TABLEBODY:
     case ROLES.TABLEFOOT:
+    case ROLES.APPLICATION:
```

Comment:
This role should generate a CompositeData instance so that it supports the global aria-* attributes and aria-activedescendant while avoiding creating a new AccessibilityObject subclass to add support for that attribute.